composer.json

```diff
@@ -14,13 +14,13 @@
     ],
     "require": {
         "php": ">=7.1",
-        "symfony/framework-bundle": "~4.2 || ~5.0",
+        "symfony/framework-bundle": "~4.2 || ~5.0 || ~6.0",
```

Comment:
same [comment for very old symfony versions](https://github.com/BedrockStreaming/GuzzleHttpBundle/pull/75#discussion_r761796423) 

=> `>= 4.4` should be good enough IMO

Reply:
✅